### PR TITLE
Some more small iterations on Calypsoify.

### PR DESCRIPTION
**Pull request by georgestephanis**
Tidying stuff, optimizing build tools, making the scss more legible and understandable.

I had the Calypsoify stuff being built previously with `-min` and `.rtl` when they really should be `-rtl` and `.min` --- my bad!

Testing Instructions:

Build it and make sure everything looks right?

Proposed changelog entry

n/a

### CODE REVIEW

File: tools/builder/sass.js [hunk 1]
```diff
@@ -60,6 +60,7 @@ gulp.task( 'sass:dops', function( done ) {
 
 function doRTL( files, done ) {
 	let dest = './inc/build',
```
Comment (dereksmart) [inline tools/builder/sass.js]: Not sure when this was changed, but the correct path should be `./_inc/` - getting a new directory after building fresh.

**Review by dereksmart — CHANGES_REQUESTED** [review]

File: tools/builder/sass.js (hunk 1 above)
Comment (georgestephanis) [reply tools/builder/sass.js]: yup, that was me in https://github.com/Automattic/jetpack/commit/aed1c420514da9474d3587d1d8930d850d4bafa6 -- fixed in 6edec5d

**Review by dereksmart — APPROVED** [review]

**Comment by dereksmart** [comment]
thank you!

**Comment by jeherve** [comment]
Cherry-picked to `branch-6.7` in 7e187a0f6f6e681f9430bed48fb89f0197bff0d6

**Pull request merged**